.github/actions/spelling/allow/occ.txt

```diff
@@ -9,6 +9,7 @@ CLA
 contentpart
 CVC
```

Comment:
For some unknowable reason the default OC themes don't include the styling for notification messages. The problem is that now the stripe styling has been moved into a separate module so it can't reference [OrchardCore.Commerce/Assets/Styles/general/_error.scss](https://github.com/OrchardCMS/OrchardCore.Commerce/blob/main/src/Modules/OrchardCore.Commerce/Assets/Styles/general/_error.scss). 

This styling shouldn't be forced from our code because it may conflict with other themes. Rather I should be handled by the theme such as #300.